.github/actions/install_python_deps/action.yml

```diff
@@ -11,5 +11,5 @@ runs:
       shell: bash
       run: |
         source ${{ inputs.ov_dir }}/setupvars.sh
-        python -m pip install ./thirdparty/openvino_tokenizers/[transformers] --pre --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
-        python -m pip install --upgrade-strategy eager -r ./samples/requirements.txt
```

Comment:
it is better to use --force-reinstall to make sure that newest versions will be installed. or --upgrade